docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3'
+
+services:
+
+  consul-agent-1: &consul-agent
```

Comment:
I don't really get it. What are all of these `agent/servers` doing?
Do we need consul agents here>
As far as I understand having 3 consul agents working as servers is what we need, right?

Reply:
It's more of an example than a real requirement, I took it from a demo `docker-compose.yml` file on their repo -> https://github.com/hashicorp/consul/blob/master/demo/docker-compose-cluster/docker-compose.yml

Do you have a suggestion on the layout of the services?

Reply:
Hmm, now that I think about it, we might actually skip the clients entirely, and use the compose file to run just the server side. What do you think?

Reply:
Sure, at least for now until we figure out if embedding the consul agents in trento is viable. Done!